dags/utils/acoustic/acoustic_client.py

```diff
@@ -157,6 +157,8 @@ def generate_report(
 
         request_body = request_body_template.render(**template_params)
 
+        print(f"DEBUG: {request_body}")
```

Comment:
Since `logging` is imported here, this can be:
```suggestion
        logging.debug(f"Request body: {request_body}")
```

Reply:
yes, the problem is that locally it's behaving as expected. I want to check in our airflow deployment to make sure old code is not running.